toolchain/check/import_ref.cpp

```diff
@@ -785,34 +799,31 @@ static auto MakeIncompleteGeneric(ImportContext& context, SemIR::InstId decl_id,
 namespace {
 // Local information associated with an imported generic.
 struct GenericData {
-  llvm::SmallVector<SemIR::InstId> bindings;
+  // TODO: Delete `GenericData` if we still don't use it once generic import is
+  // more stable.
 };
 }  // namespace
 
 // Gets a local version of the data associated with a generic.
-static auto GetLocalGenericData(ImportRefResolver& resolver,
-                                SemIR::GenericId generic_id) -> GenericData {
-  if (!generic_id.has_value()) {
-    return GenericData();
-  }
-
-  const auto& generic = resolver.import_generics().Get(generic_id);
-  return {.bindings = GetLocalInstBlockContents(resolver, generic.bindings_id)};
+static auto GetLocalGenericData(ImportRefResolver& /*resolver*/,
+                                SemIR::GenericId /*generic_id*/)
+    -> GenericData {
+  return GenericData{};
```

Comment:
```suggestion
  return {};
```

Style: "Prefer to only include the typename when required to compile" -- https://github.com/carbon-language/carbon-lang/blob/trunk/docs/project/cpp_style_guide.md

(I'm assuming it's not needed here)